packages/@lwc/engine-core/src/framework/attributes.ts

```diff
@@ -4,7 +4,6 @@
  * SPDX-License-Identifier: MIT
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
-import { assign, create } from '@lwc/shared';
```

Comment:
Using these APIs prevents Terser from tree-shaking properly. And we didn't need them.